src/turing_utilities.jl

```diff
@@ -0,0 +1,31 @@
+using Turing
+"""
+    compute_loo(chain::Chain, data, ll_fun)
```

Comment:
Docstring looks wrong -- the arguments are different.

Reply:
Fixed